wasm/generateInterface.js

```diff
@@ -281,7 +338,7 @@ function generateManifest (interfaceManifest, opts) {
     (i64.store (i32.add (get_global $sp) (i32.const ${spOffset * 32 + 8 * 2})) (i64.const 0))`
 
       if (!op.async) {
-        call += '(drop (call $bswap_m128 (i32.add (i32.const 32)(get_global $sp))))'
+        call += '(drop (i32.add (i32.const 32)(get_global $sp)))'
```

Comment:
This line is useless, it is a no-op.

In the previous case `bswap_m128` operates on memory, so it wasn't a no-op.